chart/templates/dashboards/openstack-dashboard.yaml

```diff
@@ -0,0 +1,6876 @@
+                            "thresholds": {
+                                "mode": "absolute",
+                                "steps": [
+                                    {
+                                        "color": "red"
+                                    },
+                                    {
+                                        "color": "red",
+                                        "value": 0
+                                    },
+                                    {
+                                        "color": "green",
+                                        "value": 1
+                                    }
+                                ]
+                            },
+                            "unit": "short"
+                        },
+                        "overrides": [
+                            {
+                                "matcher": {
+                                    "id": "byName",
+                                    "options": "Value"
+                                },
+                                "properties": [
+                                    {
+                                        "id": "displayName",
+                                        "value": "Status"
+                                    },
+                                    {
+                                        "id": "unit",
+                                        "value": "short"
+                                    },
+                                    {
+                                        "id": "custom.cellOptions",
+                                        "value": {
+                                            "type": "color-background"
+                                        }
+                                    },
+                                    {
+                                        "id": "custom.align",
+                                        "value": "center"
+                                    }
+                                ]
+                            },
+                            {
+                                "matcher": {
+                                    "id": "byName",
+                                    "options": "job"
+                                },
+                                "properties": [
+                                    {
+                                        "id": "custom.hidden",
+                                        "value": true
+                                    }
+                                ]
+                            },
+                            {
+                                "matcher": {
+                                    "id": "byName",
+                                    "options": "__name__"
+                                },
+                                "properties": [
+                                    {
+                                        "id": "custom.hidden",
+        
```

Comment:
This should not be here `yaook-baremetal-cluster`. Please check other places as well.